scene/gui/tree.cpp

```diff
@@ -930,7 +929,6 @@ void Tree::update_cache() {
 	cache.vseparation = get_constant("vseparation");
 	cache.item_margin = get_constant("item_margin");
 	cache.button_margin = get_constant("button_margin");
-	cache.guide_width = get_constant("guide_width");
```

Comment:
You're removing this here but not `guide_width` in the header, is this expected?

And then what about the other occurrences?
```
editor/editor_themes.cpp
707:    theme->set_constant("guide_width", "Tree", border_width);

scene/gui/tree.h
432:            int guide_width;

scene/gui/tree.cpp
933:    cache.guide_width = get_constant("guide_width");

doc/classes/Tree.xml
406:            <theme_item name="guide_width" type="int" default="2">

scene/resources/default_theme/default_theme.cpp
660:    theme->set_constant("guide_width", "Tree", 2 * scale);
```